doc/decisions/meta-decisions/use-arc42-for-architecture-overview-documents/index.md

```diff
@@ -0,0 +1,62 @@
+# Adoption of Architecture Overview Documents
+
+**Status**: Proposed
+
+**Level**: 3
```

Comment:
If this is level 3 we need to add it to the TDAG agenda to get some contributors assigned to collaborate on this. We can merge this in as Proposed at this point though.